ime/app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardBase.java

```diff
@@ -274,6 +289,13 @@ public void onFinishInput() {
         mInputSessionDisposables.clear();
         mGlobalCursorPositionDangerous = 0;
         mGlobalSelectionStartPositionDangerous = 0;
+
+        if (mInputViewContainer != null) {
```

Comment:
If you move this logic to `onFinishInputView`, you can be sure that the view was shown prior.

Reply:
works like a charm! thanks for the suggestion!